extensions/wikia/CloseMyAccount/CloseMyAccountSpecialController.class.php

```diff
@@ -174,6 +174,10 @@ public function reactivate() {
 			return;
 		}
 
+		// call to User::newFromConfirmationCode() is basically a token check
+		// tell CSRFDetector that we're fine here (PLATFORM-2206)
+		\Wikia\Security\CSRFDetector::onUserMatchEditToken();
```

Comment:
It's the first case when we need to call this method outside of .`CSRFDetector` Let's wait for a second one.  Sounds fine?